src/wifes_calib.py

```diff
@@ -793,36 +793,55 @@ def derive_wifes_telluric(cube_fn_list,
             ex_data = numpy.loadtxt(extract_in_list[i])
             obs_wave = ex_data[:,0]
             obs_flux = ex_data[:,1]
+        
+        # Filtering nan values in case of missing flux values
+
+        filter_nan = ~numpy.isnan(obs_flux)
+        obs_wave_nan = obs_wave[filter_nan]  
+        obs_flux_nan = obs_flux[filter_nan]
+        
+
+        print("OBS_WAVE_NAN", obs_wave_nan)
+        print("OBS_FLUX", obs_flux_nan)
+        print("O2_tell_bands", O2_telluric_bands)
+        print("H2O_tell_bands", H2O_telluric_bands)
```

Comment:
Best not to leave debug output in. If this is useful in general, we might need to add a logger, or a verbose flag?